.github/workflows/ci.yml

```diff
@@ -0,0 +1,29 @@
+name: CI
+
+on: push
+
+permissions:
+  contents: read
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    continue-on-error: ${{ contains(fromJSON('["2.5.0", "ruby-head", "jruby-9.1", "jruby-head"]'), matrix.ruby-version) }}
```

Comment:
I would just set this to true/false

Reply:
Yeah, that would be much simpler. I wanted to stay close to the old `travis.yml` as much as possible i.e. `ruby-head`, `jruby-9.1` and `jruby-head` are optional steps.